package.json

```diff
@@ -45,8 +56,8 @@
     "babel-core": "^6.26.3",
     "eslint": "^6.6.0",
     "flow-bin": "^0.98.0",
-    "react": "16.8.6",
-    "react-native": "0.60.0"
+    "react": "^16.8.6",
+    "react-native": "^0.60.0"
```

Comment:
Hey @jdnichollsc - Just a quick question: Did you have any issues with the fixed versions of `react` and `react-native` here? Thanks!

Reply:
these are dev dependencies, why do you need fixed versions?

Reply:
Because one version of `react-native` is always tightly coupled with exactly one version of `react`. They can't go out of sync, or things will likely break. That's the reason a new project generated with the React Native CLI will have these versions fixed (no `^` in front). The same applies to most library projects that declare these as `devDependencies`.

Since `react` releases independently, say for example `16.10.0`, it would upgrade it here, which is what you don't want.. Any upgrade to react or react-native should

1. be _explicit_, and
2. be atomic (together with its `react` dependency)

I created a PR here: #124 
Please consider it :)